README.md

```diff
@@ -20,3 +20,93 @@ have to start from scratch to fill it with content.
+messages sent by the contact form, blog articles etc. Clicking on each category
+opens up information about that thing ie. users and gives you the ability to
+change them.
+
+Admins shouldn't need to change much using the page menu. It is powerful and
+capable of deleting and creating new pages which you do not want to do
+accidentally. There are two main cases in which you may wish to use this menu:
+
+1. If you made a change to a page and want to see what the page looked like
+previously. To do this click on 'Page Versions':
+
+<img src="https://user-images.githubusercontent.com/16775804/35848458-246cd034-0b16-11e8-84ab-8513f384ff43.png" width=150px />
+
+Here you can see recent changes to the page and revert back to them if needs be.
+Note: be careful and pay attention when reverting changes.
+
+<img src="https://user-images.githubusercontent.com/16775804/35848520-5a3dbc64-0b16-11e8-9c37-fa4301e80571.png" width=800px />
+
+2. If you want to create a new blog post. Start on the main blog page and click
+the menu, you will see the option to create a new article:
+
+<img src="https://user-images.githubusercontent.com/16775804/35848737-0d83301a-0b17-11e8-944d-9f83e0af95ad.png" width=200px />
+
+Then fill in the fields in the three tabs.
+
+The term 'slug' is part of the url that will appear in the top of the browser,
+so there must not be any spaces in it. The slug will be automatically generated
+using your blog title and so you don't need to change it if you don't wish. A
+long, descriptive slug is fine as it's good for SEO.
+
+You must change published to 'yes' if you want the post to be visible on the site.
+
+### Editing content
+
+A lot of the content on the site is editable but not all of it. To see if a
+piece of text is editable, when you're logged in as an admin try double clicking
+on the field. If it is editable you will see a dotted box appear around it:
+
+<img src="https://user-images.githubusercontent.com/16775804/35848910-b7c528a8-0b17-11e8-8e47-6f31a1023602.png" width=200px />
+
+Then click on the text and you can edit it.
+
+You may also notice these symbols showing when you move around the site:
+
+<img src="https://user-images.githubusercontent.com/16775804/35849016-146471d6-0b18-11e8-91a1-f0a9b040c330.png" width=200px />
+
+They are also a sign that something is editable. Rather than deleting something
+by clicking the trash can icon, try double clicking the content to get the dotted
+box. If you do accidentally click the trash icon and you want to recreate what
+you deleted look where it used to be for something like this:
+
+<img src="https://user-images.githubusercontent.com/16775804/35849102-5b3734fe-0b18-11e8-8306-261a66e8e079.png" width=400px />
+
+Click on that and then enter in the text that was there before. You should not
+need to delete the field and add it again, normally just double click it to see
+the dotted box.
+
+### Writing a blog article
```

Comment:
Have just seen this section 👍 
Might be good to have the whole bit about Creating a blog article from scratch altogether or at least referencing the above where it begins.